### ISSUE TITLE
Error in ErrorGroup component

### ISSUE THREAD

**Issue by cschloer** [frictionlessdata/goodtables-ui#34]
# Overview

I'm not sure if this is on my end or on the end of the React component.

My code looks something like this:

```
import goodtablesUI from 'goodtables-ui/lib';

...
  return (
    <div>
    ...
        <goodtablesUI.Report
          report={report}
        />
    ...
    </div>
  )
...
```

It works fine for reports that are valid. If there is a non valid report, it throws `The above error occurred in the <ErrorGroup> component:`. 
```
Uncaught Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.
```

The same error occurs if I format the code to look more like the documentation and use ReactDOM:
```
const report = '<YOUR-REPORT>'
const element = document.getElementById('report')
ReactDOM.render(<goodtablesUI.Report report={report} />, element)
```

---

Please preserve this line to notify @roll (lead of this repository)

**Comment by roll** [comment]
Thanks! I'll investigate

**Comment by roll** [comment]
Hi @cschloer,

I wasn't able to reproduce it with an invalid report (it's also deployed to https://frictionlessdata.github.io/goodtables-ui/report.html)

I think it must be one of them:
- 1. You might have mismatching versions of React and the renderer (such as React DOM)
- 3. You might have more than one copy of React in the same app

Actually, I think that problem with the import (we need to import from `src` for native React components):

```js
import goodtablesUI from 'goodtables-ui/src';
```

Please re-open if it didn't help

**Comment by roll** [comment]
> Actually, I think that problem with the import (we need to import from src for native React components)

No, sorry it's not correct - we need to export from `lib`

**Comment by roll** [comment]
And now I was able to reproduce it :smiley:

**Comment by roll** [comment]
So I'm getting this errors if I export from `goodtables-ui` but it works fine if I export from `goodtables-ui/lib`:

- I started a new Create React App
- installed `goodtables-ui`
- updated `src/App.js` (also adding `report.json` to `src`)

```js
import report from "./report.json";
import goodtablesUI from "goodtables-ui/lib";
import React from "react";
import logo from "./logo.svg";
import "./App.css";

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
      <goodtablesUI.Report report={report} />
    </div>
  );
}

export default App;
```

It works fine.

Please re-open if you can't find the problem on the Laminar side (sorry for a los of messages)

**Comment by roll** [comment]
Also I would try exporting exact components `import {Report} from `goodtables-ui/lib/components/Report`

**Comment by cschloer** [comment]
```
If you see more than one React, you’ll need to figure out why this happens and fix your dependency tree. For example, maybe a library you’re using incorrectly specifies react as a dependency (rather than a peer dependency). Until that library is fixed, Yarn resolutions is one possible workaround.
```

Is it possible goodtables is incorrectly specifying React as a dependency? I see two versions of react when I run `npm ls react` and the second one is coming from goodtables.

**Comment by cschloer** [comment]
Can't reopen for some reason but consider it reopened :) @roll

**Comment by roll** [comment]
You're right. It was somehow not in `peerDependencies`.

**Comment by roll** [comment]
Fixed in a0953d0f9c56377562c2981ce06b4bed4f5fd15f

@cschloer 
Please try `goodtables-ui@2.1`

**Comment by cschloer** [comment]
thanks @roll. works great :)
